From 416401b4396922da7695ce8a492da4ab8f8317f6 Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Mon, 13 Jun 2016 23:39:05 +0200
Subject: [PATCH] Disable session management for kactivitymanagerd.

I don't actually need 22 instances of this program running at the same time :-)
---
 src/service/Application.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/service/Application.cpp b/src/service/Application.cpp
index 7cddf67..0ca0b72 100644
--- a/src/service/Application.cpp
+++ b/src/service/Application.cpp
@@ -310,6 +310,9 @@ namespace  {
 
 int main(int argc, char **argv)
 {
+    // Disable session management for this process
+    qunsetenv("SESSION_MANAGER");
+
     Application application(argc, argv);
     application.setApplicationName(QStringLiteral("ActivityManager"));
     application.setOrganizationDomain(QStringLiteral("kde.org"));
-- 
1.9.5

